src/Microsoft.AspNet.Razor.Test.Sources/TagHelperAttributeDescriptorComparer.cs

```diff
@@ -38,15 +38,17 @@ public bool Equals(TagHelperAttributeDescriptor descriptorX, TagHelperAttributeD
 
         public int GetHashCode(TagHelperAttributeDescriptor descriptor)
         {
-            return HashCodeCombiner.Start()
-                .Add(descriptor.IsIndexer)
-                .Add(descriptor.Name, StringComparer.Ordinal)
-                .Add(descriptor.PropertyName, StringComparer.Ordinal)
-                .Add(descriptor.TypeName, StringComparer.Ordinal)
-                .Add(descriptor.IsStringProperty)
-                .Add(TagHelperAttributeDesignTimeDescriptorComparer.Default.GetHashCode(
-                    descriptor.DesignTimeDescriptor))
-                .CombinedHash;
+            var hashCodeCombiner = HashCodeCombiner.Start();
+
```

Comment:
Suggest consistency in these `GetHashCode()` implementations would be nice. Looks like w/o the blank line is more common.